### DISCUSSION TITLE
[V5] VSCode Type Declarations

### DISCUSSION THREAD

**Discussion by ap1969** [adonisjs/core#3048] [Help]
Hi all,
I'm new to both vscode and adonisjs, and getting frustrated with loads of entries in the problem view of VSCode saying:

`Cannot find module '@ioc:Adonis/Lucid/Database' or its corresponding type declarations.`

Anyone know how to get rid of these?

Thanks
Andy

**Comment by McSneaky** [comment]
Have you installed them?

If yes, then restart or reload VSCode

If you haven't installed them, then [install em](https://docs.adonisjs.com/guides/database/introduction) and after that restart or reload VSCode
